tools/seq_filter_by_id/seq_filter_by_id.xml

```diff
@@ -1,8 +1,7 @@
-<tool id="seq_filter_by_id" name="Filter sequences by ID" version="0.2.3">
+<tool id="seq_filter_by_id" name="Filter sequences by ID" version="0.2.5">
     <description>from a tabular file</description>
     <requirements>
         <requirement type="package" version="1.64">biopython</requirement>
-        <requirement type="python-module">Bio</requirement>
```

Comment:
@bgruening @jmchilton do you know if ``<requirement type="python-module">`` still has any effect, or who would know? See also Tool XSD documentation https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-requirements-requirement which is vague on this. Thanks!